lib/shopify_app/sessions_concern.rb

```diff
@@ -87,5 +90,16 @@ def return_address
       session.delete(:return_to) || main_app.root_url
     end
 
+    def perform_after_install_actions
+      return unless ShopifyApp.configuration.enable_after_install_actions
```

Comment:
This is actually pretty similar to how we set the object for the SessionRepository https://github.com/Shopify/shopify_app/blob/master/example/config/initializers/shopify_session_repository.rb

Come to think of it why isn't the SessionRepository configured in the main ShopifyApp initializer? That would be a good follow up :)